recipes/perl-module-build/meta.yaml

```diff
@@ -11,17 +11,17 @@ source:
    # - fix.patch
 
 build:
-  number: 2
+  number: 3
   skip: False
 
 requirements:
   build:
     - gcc
-    - perl-threaded >=5.22.0
+    - perl
```

Comment:
@daler I hope removing the minimum version is fine with you, the [Travis build](https://travis-ci.org/bioconda/bioconda-recipes/jobs/196708452#L736) fails without it.